src/main/java/com/example/glovo/repository/OrderRepository.java

```diff
@@ -0,0 +1,46 @@
+package com.example.glovo.repository;
+
+import com.example.glovo.dto.OrderDto;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class OrderRepository {
+    private final Map<Integer, OrderDto> orders = new HashMap<>();
```

Comment:
try to avoid Integer as a key as you may lose the data because of collisions. consider using String